### DISCUSSION TITLE
5주차 미팅

### DISCUSSION THREAD

**Discussion by bluedog129** [GunGonGamLee/ft_transcendence#133] [General]
1. 개발 계획 잡기 (팀 전체, 개인) 
 - 이번 주는 ~~ 까지 하겠다 
 - 다음 주는 ~~ 을 시작하자

**Comment by nyj001012** [comment]
- 이번 주
  - 사용자 전적 페이지 완료

- 다음 주
  - 소켓

**Comment by bluedog129** [comment]
- 이번 주 목표 
   사용자 지정 게임 방 목록 페이지 완료
- 다음주
   친구창 모달, 소켓 (도와주세여)

**Comment by sejoonkimmm** [comment]
* 이번주 목표
친구 추가 / 신청 / 거절 / 목록 구현 및 test code 구현

* 다음주 목표
ELK 다시 뜯어서 클러스터 맥에서 원하는대로 log 잘 찍힐 수 있도록... 이게 다음주 안에 될까요? 파이팅

**Comment by LWJ0513** [comment]
### 이번주 목표
게임방 생성, 참가, 리스트 API 구현

### 다음주 목표
리스트 API 마저 구현
